src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -200,8 +200,8 @@ def get_profile(
                 f"A type checker was asked to check a type it did not have registered, {exc}"
             )
         except jsonschema.exceptions.UnknownType as exc:
-            logger.error(f"Unknown type is found in schema_json, exc")
-            raise jsonschema.exceptions.UnknownType(f"Unknown type is found in schema_json, exc")
+            logger.error(f"Unknown type is found in schema_json, {exc}")
+            raise jsonschema.exceptions.UnknownType(f"Unknown type is found in schema_json, {exc}")
```

Comment:
Good catch!

Not sure if we should do a quick search across the board to make sure that we are interpolating/inserting the value of the error instead of the literal string of the variable name